frontend/src/components/SiteHeader.tsx

```diff
@@ -24,13 +24,13 @@ export default class SiteHeader extends React.Component<
             </Row>
             <Row>
               <Col>
-                <Link to="/login"> Login </Link>
+                <Link to="/login"> <h3> Login </h3></Link>
```

Comment:
@vong98 In terms of heading hierarchy for SEO these should be h2s.

Not a big deal since we'll never release but good standard to try and keep even for just for making styling easier.
https://www.hobo-web.co.uk/headers/

Reply:
Changed! I made h2 a bit smaller though (20pt) as 30pt was looking just slightly too big next to uBuy.